tests/phpunit/bootstrap.php

```diff
@@ -9,6 +9,7 @@
 function _manually_load_plugin() {
 	$plugin_dir = dirname( dirname( dirname( dirname( __FILE__ ) ) ) ) . '/';
 	require $plugin_dir . 'woocommerce/woocommerce.php';
+	update_option( 'woocommerce_default_country', 'US:CA' );
```

Comment:
This doesn't feel like quite the right place to set the test store's default country, but I wasn't sure where else to do it. Without this, since the gateway doesn't load at all for unsupported countries, most tests would fail.